drivers/can/Kconfig.mcux

```diff
@@ -0,0 +1,21 @@
+# Kconfig.mcux - MCUX CAN configuration options
+
+#
+# Copyright (c) 2019 Vestas Wind Systems A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config CAN_MCUX_FLEXCAN
+	bool "MCUX FlexCAN driver"
+	depends on HAS_MCUX_FLEXCAN && CLOCK_CONTROL
```

Comment:
You can use default y here to automatically select FlexCAN when CAN driver is selected

Reply:
Yes, but the standard way of handling this (at least for the NXP Kinetis SoCs and drivers) is to enable it conditionally in `Kconfig.defconfig` for the given SoC. I'd like to keep this in line with that.